recipes/gwsumm/meta.yaml

```
@@ -0,0 +1,105 @@
+{% set name = "gwsumm" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0dd3ec91cf51fe868e2f77065c15816c8e1ee87e4f0d3fdef03b64b340d7a19f
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - jsmin
+    - libsass
+    - pip
+    - python
+    - setuptools
+  run:
+    - python >=2.7
+    - enum34  # [py<34]
+    - python-dateutil
+    - lxml
+    - numpy >=1.10
+    - scipy >=1.2.0
+    - matplotlib >=2.0
+    - astropy >=1.2.1
+    - lalsuite
+    - gwtrigfind
+    - gwdatafind
+    - lscsoft-glue >=1.60.0
+    - ligo-segments
+    - pygments
+    - MarkupPy
+    - gwdetchar >=0.5.1
+    - gwpy >=0.14.2
+    - configparser  # [py<36]
+
+test:
+  requires:
+    - pytest
+    - mock  # [py<30]
+  imports:
+    - gwsumm
+    - gwsumm.archive
+    - gwsumm.channels
+    - gwsumm.config
+    - gwsumm.data
+    - gwsumm.data.coherence
+    - gwsumm.data.mathutils
+    - gwsumm.data.range
+    - gwsumm.data.spectral
+    - gwsumm.data.timeseries
+    - gwsumm.data.utils
+    - gwsumm.globalv
+    - gwsumm.html
+    - gwsumm.html.bootstrap
+    - gwsumm.html.html5
+    - gwsumm.html.static
+    - gwsumm.io
+    - gwsumm.mode
+    - gwsumm.segments
+    - gwsumm.state
+    - gwsumm.state.all
+    - gwsumm.state.registry
+    - gwsumm.triggers
+    - gwsumm.units
+    - gwsumm.utils
+  commands:
+    # run unit tests
+    - python -m pytest --pyargs {{ name }}
+    # check scripts run ok
+    - gw_summary --help
+    - gw_summary day --help
+    - gw_summary week --help
+    - gw_summary month --help
+    - gw_summary gps --help
+    - gw_summary_pipe --help
+    - gwsumm-plot-guardian --help
+    - gwsumm-plot-triggers --help
+    # check HTML files were built and installed
+    - test -f ${SP_DIR}/gwsumm/html/static/gwsumm.min.css
+    - test -f ${SP_DIR}/gwsumm/html/static/gwsumm.min.js
+
+about:
+  home: https://github.com/gwpy/gwsumm
+  license: GPLv3
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```